unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Animator/DCLAnimator.cs

```diff
@@ -43,6 +43,8 @@ public class DCLAnimationState
         Dictionary<string, AnimationClip> clipNameToClip = new Dictionary<string, AnimationClip>();
         Dictionary<AnimationClip, AnimationState> clipToState = new Dictionary<AnimationClip, AnimationState>();
 
+        public override string componentName => "animator";
```

Comment:
We should always uses camelCase or Snake, so this  should be Animator or the rest should be audioSource